### DISCUSSION TITLE
Idea for new xml tag podcast:video to support video podcasts

### DISCUSSION THREAD

**Discussion by killianogorman** [theDanielJLewis/podcasting2.org#8] [Ideas]
# Video
With the growing landscape of podcasts. Many creators have turned to the also provide a video option. As this has become more popular to do. I think podcast 2.0 should feature a `<podcast:video>` tag. This could allow for podcast apps to offer users to download and view their favourite podcasts without having to the app. Below is a possible implementation of the `<podcast:video>` tag.
## Attributes
- url (required): URL of the podcast video
- type (required): MIME type of video such as `application/vnd.apple.mpegurl` `video/mp4`
- quality: The file's quality. eg: 720p
```xml
<podcast:video>
```
Example 
```xml
<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0" xmlns:media="" xmlns:atom="" xmlns:ppg="" xmlns:itunes="">
	<channel>
		<title></title>
		...
		<item>
			<title></title>
			...
			<podcast:video url="https://example.com/episode1/video.m3u8" type="application/vnd.apple.mpegurl" />
			<podcast:video url="https://example.com/episode1/video.m3u8" type="video/mp4" quality="1080" />
		</item>
	</channel>
</rss>
```

**Comment by jamescridland** [comment]
This is a great idea - thanks so much for it!

There are two things the existing specification can help us with - first the standard enclosure tag can have video in it (using the mimetypes as you’ve done); and secondly, the ‘podcast:alternateEnclosure’ Podcasting 2.0 tag allows a podcaster to specify more than one type of file as an enclosure. So you can specify an audio version and a video version of the same thing - allowing podcast apps to choose whatever the listener (or viewer) has asked for. That even lets you link over to YouTube videos if you want to do that. 

 So video is already well catered for here; all we need to do is to encourage more podcasters to make feeds with those alternate enclosures in.

Always good to get some good ideas, so thank you!
